runtime/src/main/java/org/corfudb/runtime/object/CorfuCompileProxy.java

```diff
@@ -92,6 +94,26 @@
      */
     Map<Long, Object> upcallResults;
 
+    /**
+     * Metrics: meter (counter), histogram
+     */
+    private static final Timer timerAccess = CorfuRuntime.getMetrics().timer(
+            CorfuRuntime.getMpObj() + "access");
+    private static final Timer timerLogWrite = CorfuRuntime.getMetrics().timer(
+            CorfuRuntime.getMpObj() + "log-write");
+    private static final Timer timerTxn = CorfuRuntime.getMetrics().timer(
+            CorfuRuntime.getMpObj() + "txn");
+    private static final Timer timerUpcall = CorfuRuntime.getMetrics().timer(
+            CorfuRuntime.getMpObj() + "upcall");
+    private static final Counter counterAccessOptimistic = CorfuRuntime.getMetrics().counter(
+            CorfuRuntime.getMpObj() + "access-optimistic");
+    private static final Counter counterAccessLocked = CorfuRuntime.getMetrics().counter(
+            CorfuRuntime.getMpObj() + "access-locked");
+    private static final Counter counterTxnRetry1 = CorfuRuntime.getMetrics().counter(
+            CorfuRuntime.getMpObj() + "txn-retry1");
+    private static final Counter counterTxnRetryN = CorfuRuntime.getMetrics().counter(
+            CorfuRuntime.getMpObj() + "txn-retryN");
+
```

Comment:
again, none of these need to be static. Calling metricsRegistry.counter( some-name ) does the work for you.